roles/openshift_service_catalog/tasks/start.yml

```diff
@@ -1,6 +1,21 @@
 ---
-# TODO: abstract me into a relatively generic task
-- name: Verify that the catalog api server is running
+- name: Wait for API Server rollout success
+  command: >
+      {{ openshift_client_binary }} rollout status --config=/etc/origin/master/admin.kubeconfig -n kube-service-catalog ds/apiserver
```

Comment:
Does this block until the rollout is complete or will it return immediately if incomplete? If the latter we need to add some retries so that it checks multiple times with a reasonable upper limit before giving up.

Reply:
> Does this block until the rollout is complete or will it return immediately if incomplete? 

It blocks until complete or an error.   I've verified this via several updates in multi master installs, and also found other components using it in the same manner (I think it it was openshift console)

Reply:
Probably should have some retries just for network flakiness sake.

Reply:
@michaelgugino So adding something like this to each `rollout status` check?
```
  until: apiserver_rollout_status.rc == 0
  retries: 5
  delay: 10
```

Reply:
Yes, that should be suitable IMO.

Reply:
Ok thanks @jboyd01, lgtm. If it's easy to distinguish between rollout problems and the API being down momentarily I guess we should try to do that.

Reply:
I added the retries and just did another verification pass.  I didn't try to induce any network or other failure to get the retries to fire, but this worked for me.  Thanks for the reviews and discussion.